server/socket/emitProjects/index.js

```diff
@@ -1,11 +1,57 @@
+import _ from 'lodash'
+import moment from 'moment'
 import { asyncRepeat } from 'common/utilities/asyncRepeat'
 import circleCI from 'common/circleCI'
 import { socket } from '../'
 
+const sortableTimeFormat = 'YYYYMMDDHHmmss'
+
+const getRecentlyBuiltBranches = branches => {
+  return _.chain(branches)
+    .map((branch, name) => ({ ...branch, name: name }))
+    .sortBy(branch => {
+      const latestBuild = _.first(branch.recent_builds)
+      return latestBuild ? moment(latestBuild.pushed_at).format(sortableTimeFormat) : 0
+    })
+    .reverse()
+    .slice(0, 3)
```

Comment:
could the slice be a `.take` or a `.drop` which I find a bit easier to think about.